src/translations/components/LanguageSwitcher.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { locales } from "../config";
+import useTranslation from "../hooks/useTranslations";
+import Link from "next/link";
+
+const LocaleSwitcher: React.FC = () => {
```

Comment:
dummy component to test language switching - we can remove it later / adjust accordingly to our needs